Lib/http/client.py

```diff
@@ -265,7 +269,7 @@ def _read_status(self):
         if len(line) > _MAXLINE:
             raise LineTooLong("status line")
         if self.debuglevel > 0:
-            print("reply:", repr(line))
+            _log.info("reply: {}".format(repr(line)))
```

Comment:
_logging_ first argument support old format string, so `_log.info("reply: %s", repr(line))` would work. There is mixed calls of `{}".format(foo` with `%s", foo`, wouldn't be better to stick with one?

Reply:
> wouldn't be better to stick with one?

I agree - `%` should work fine and a lot of logging uses %-formatting rather than `.format()` due to its age.

Reply:
There is another reason: tools like Sentry use message pattern as a key for logs grouping.
Adding pre-formatted messages to Sentry is very ineffective.

Reply:
Ah, sorry!

I think I came back to this later and followed the same style as for other Python projects I was involved with.

Fixing this!